models/users.js

```diff
@@ -541,9 +545,29 @@ const getUsersBasedOnFilter = async (query) => {
 
     return filteredUsers.filter((user) => !user.roles?.archived);
   }
+
   return [];
 };
 
+const getUsersWithOnboardingState = async (filteredUserDocs, stateItems, time) => {
```

Comment:
can we rename it to `getUsersWithOnboardingStateInRange` since it returns the users with onboarding state in the range

Reply:
fixed